src/Form.php

```diff
@@ -49,6 +49,18 @@ public function addField(string $name, string $content, ?string $contentType = n
         $this->fields[] = new FormField($name, BufferedContent::fromString($content, $contentType));
     }
 
+    /**
+     * Adds each member of the array as an entry for the given key name. Array keys are persevered.
+     *
+     * @param array<string> $fields
+     */
+    public function addFields(string $name, array $fields): void
```

Comment:
Based on our usual naming, I'd expect this to add a repeated field, not nested fields based on the array keys.

Reply:
I agree. `addArrayField`? Or something else?

Reply:
Maybe `addNestedFields`?

Reply:
Works for me. Updated.